test/rpc_test.py

```diff
@@ -1179,6 +1188,59 @@ def test_rref_str(self):
             "UserRRef(RRefId = {0}({1}, 1), ForkId = {0}({1}, 2))".format(id_class, self.rank)
         )
 
+    @dist_init
+    def test_rref_context_debug_info(self):
+        if not dist.is_initialized():
+            dist.init_process_group(
+                backend="gloo",
+                init_method=self.init_method,
+                rank=self.rank,
+                world_size=self.world_size,
+            )
+
```

Comment:
add a `dist.barrier()` here to make sure all pg initialization are completed?

Reply:
This should be fine as `init_process_group` itself is a barrier.